### ISSUE TITLE
Convert Usages

### ISSUE THREAD

**Issue by jmccrae** [elexis-eu/tei2ontolex#8]
Convert the `<usg>` elements into OntoLex

```xml
<sense n="1">
 <def xml:lang="de">Frucht des Pflaumenbaums</def>
 <def xml:lang="en">fruit of the plum tree</def>
</sense>
<sense n="2">
 <usg type="register" norm="colloquial">ugs.</usg>
 <def xml:lang="de">Pflaumenbaum</def>
 <def xml:lang="en">plum tree</def>
</sense>
```

As OntoLex with Turtle

```turtle
ontolex:sense [
  skos:definition "Frucht des Pflaumenbaums"@de ;
  skos:definition "fruit of the plum tree"@en
] , [
   skos:definition "Pflaumenbaum"@de ;
   skos:definition "plum tree"@en ;
   lexinfo:register [ rdf:value "ugs."@de , "colloquial" ]
 ] .
```

**Comment by laurentromary** [comment]
@jmccrae to make sure: can you provide the XML serialization of ` lexinfo:register [ rdf:value "ugs."@de , "colloquial" ]`

**Comment by jmccrae** [comment]
Something like
```
<lexinfo:register> 
  <rdf:Description>
    <rdf:value xml:lang="de">ugs.</rdf:value>
    <rdf:value>colloquial</rdf:value>
  </rdf:Description>
</lexinfo:register>

**Comment by laurentromary** [comment]
I need lexinfo equivalents for geographic, domain, attitude, normativity, meaningType, textType, hint.

**Comment by jmccrae** [comment]
See https://github.com/elexis-eu/tei2ontolex/blob/master/tests/test7.ttl for the list of equivalents

**Comment by jmccrae** [comment]
NB these are all new ones that I added to LexInfo 3.0

**Comment by laurentromary** [comment]
Implemented. There remains a doubt as to whether I should use socioCultural or register (which are equivalent) => which one is the official lexinfo category?

**Comment by jmccrae** [comment]
The `register` property is connected to the [fixed set of categories defined by ISO](https://en.wikipedia.org/wiki/Register_(sociolinguistics)#ISO_standard)

We should use open `socioCultural` property

**Comment by laurentromary** [comment]
Implemented in #3730375

**Comment by jmccrae** [comment]
Great, this works now
